components/cards/default/card.hbs

```diff
@@ -0,0 +1,19 @@
+<div class="section--bg-white demo-section">
+  <div class="content-block">
+    <img
+      class="content-block__image"
+      src="https://via.placeholder.com/535x301"
+      alt=""
+    />
+    <date class="content-block__date">
+      <span class="content-block__date-month">Jun</span>
+      <span class="content-block__date-day">10</span>
+    </date>
+    <p class="content-block__image-tag">Digital Webinars</p>
+    <h4 class="content-block__title">RSVP: This is ...</h4>
+    <p class="content-block__text">Lorem ipsum dolor, sit amet consectetur
+      adipisicing elit. Voluptate sit id harum aliquid ea.
+    </p>
+    <a class="button--medium-primary">Learn More</a>
```

Comment:
Let's wrap this in something like `<div class="content-block__lower">`. That will make this readily flexible when we get to the 50/50 side-by-side version.